optashift-employee-rostering-server/src/main/java/org/optaplanner/openshift/employeerostering/server/solver/WannabeSolverManager.java

```diff
@@ -108,4 +115,40 @@ public Roster getRoster(final Integer tenantId) {
         Solver<Roster> solver = tenantIdToSolverMap.get(tenantId);
         return solver == null ? null : solver.getBestSolution();
     }
+
+    public Map<Shift, IndictmentView> getShiftIndictmentMap(Integer tenantId) {
```

Comment:
SolverManager can't return "IndictmentView", because the later isn't optaplanner-api and SolverManager will be pushed upstream into optaplanner-core someday (in some form, it might be ripped apart).
Extract the logic to create IndictmentViews into the RosterService.